app/models/snap_application.rb

```diff
@@ -65,6 +65,18 @@ def residential_address
     addresses.where.not(mailing: true).first || NullAddress.new
   end
 
+  def zip
+    residential_address.zip
+  end
+
+  def receiving_office
```

Comment:
thoughts on putting this below `receiving_office_name` so that the methods in this file are in dependence order? (methods are above other methods that they depend on). This is something I picked up from Avdi Grimm's ruby tapas!

Reply:
Nice call! I wish there was a rubocop for that!